### PR TITLE
GA/Authorization SP-13/SP-7/2

### PR DESCRIPTION
**Pull request by alfiyaGumar**

### CODE REVIEW

File: tests_advance/authorization.spec.js [hunk 1]
```diff
@@ -28,4 +29,32 @@ test.describe('Negative tests for Authorization process', () => {
             await expect(loginPage.emailAddressError).toHaveText('Required');
         });
     });
+
+    test('SP-13/SP-7/2 | Verify non-successful authorization of Business user in case of incorrect email address.', async ({
```
Comment (Elgrueber) [inline tests_advance/authorization.spec.js:33]: What is incorrect in email address? Why email address is incorrect? I did this test as parametrized and add in the name of test - what exactly incorrect in the email address. If you don't want to make parametrized test - I suppose it is still need to write the reason why e-mail address is incorrect

Comment (alfiyaGumar) [reply tests_advance/authorization.spec.js:33]: This is simply an incorrect email. There is no registration with this email. In this case, a toast message appears stating that either the email or the password is incorrect, that they do not match each other. Also, I wanted to create a parameterized test with an invalid email next time (without @, just with special characters and with a space).

Comment (Elgrueber) [reply tests_advance/authorization.spec.js:33]: ok. but i would write in the name of test invalid login and valid password (for example)

Comment (verafes) [reply tests_advance/authorization.spec.js:33]: suggestion: Failed login with invalid email and valid password

Comment (alfiyaGumar) [reply tests_advance/authorization.spec.js]: Done!

File: tests_advance/authorization.spec.js [hunk 2]
```diff
@@ -28,4 +29,29 @@ test.describe('Negative tests for Authorization process', () => {
             await expect(loginPage.emailAddressError).toHaveText('Required');
         });
     });
+
+    test('SP-13/SP-7/2 | Verify failed login with invalid email and valid password.', async ({ page, loginPage }) => {
```
Comment (Elgrueber) [inline tests_advance/authorization.spec.js:33]: SP1/SP2/1 - please follow instruction how to name tests

Comment (alfiyaGumar) [reply tests_advance/authorization.spec.js]: Done

**Pull request merged**